test/test_scanner.py

```diff
@@ -740,6 +740,22 @@ def _file_test(self, url, filename, package, version):
                     "CVE-2005-2096"
                 ],
             ),
+            (
+                "test-xml2-2.9.3.out",
```

Comment:
Group with at least one of the existing xml2 tests, or maybe move them all to be together in this PR please.

Reply:
Terri, I have grouped all the xml2 tests together.